mypy/checker.py

```diff
@@ -976,13 +977,17 @@ def check_func_def(self, defn: FuncItem, typ: CallableType, name: Optional[str])
                                 ctx = typ
                             self.fail(message_registry.FUNCTION_PARAMETER_CANNOT_BE_COVARIANT, ctx)
                     if typ.arg_kinds[i] == nodes.ARG_STAR:
-                        # builtins.tuple[T] is typing.Tuple[T, ...]
-                        arg_type = self.named_generic_type('builtins.tuple',
-                                                           [arg_type])
+                        arg_type = get_proper_type(arg_type)
+                        if not isinstance(arg_type, ParamSpecType):
```

Comment:
I think you should update the proper_plugin to not require this. `ParamSpecType` can never by a target of a type alias. (Note `TypeVarType` is already excluded).

Reply:
A good idea! Done. Also removed various now-redundant `get_proper_type` calls.